packages/builder/src/components/automation/SetupPanel/RowSelectorTypes.svelte

```diff
@@ -195,20 +253,41 @@
   <svelte:component
     this={isTestModal ? ModalBindableInput : DrawerBindableInput}
     panel={AutomationBindingPanel}
-    value={value[field]}
-    on:change={e => onChange(e, field)}
+    value={fieldData}
+    on:change={e =>
+      onChange({
+        row: {
+          [field]: e.detail,
+        },
+      })}
     type="string"
     bindings={parsedBindings}
     allowJS={true}
     updateOnChange={false}
     title={schema.name}
+    autocomplete="off"
   />
 {/if}
 
 <style>
-  .attachment-field-spacing,
-  .json-input-spacing {
-    margin-top: var(--spacing-s);
-    margin-bottom: var(--spacing-l);
+  .attachment-field-spacing {
+    border: 1px solid var(--spectrum-global-color-gray-400);
+    border-radius: 4px;
+    padding: var(--spacing-s);
+  }
+
+  .field-wrap.json-field {
```

Comment:
NAB: Is there a way to make the code-editor resizeable? At a glance I don't see any obvious props that make it possible, but perhaps you know more about it than I do.

The reason I asked, is that 80px is a little titchy IMO, it'd be hard to write or view any complex JSON.

![Screenshot 2024-07-03 at 12 43 00](https://github.com/Budibase/budibase/assets/12501626/03b7f3cb-4fad-4fc6-9b2d-e48d560a3045)

If resizing isn't possible, would bumping the size up make sense?

Reply:
At the minute no, it behaves in the same way as the old JSON editor. It allows you to quickly glance over your content and if they want more screen real-estate, they can click on the lightning bolt.

Reply:
I bumped the default height by another 40 pixels and be we should definitely look into making it resizable.